tests/integrationtest/r/planner/core/indexmerge_path.result

```diff
@@ -714,3 +714,28 @@ id	estRows	task	access object	operator info
 IndexMerge_7	1.00	root		type: union
 ├─IndexRangeScan_5(Build)	1.00	cop[tikv]	table:t, index:ibb(cast(json_extract(`b`, _utf8mb4'$.b') as signed array))	range:[10,10], keep order:false, stats:partial[ibb:missing, b:unInitialized]
 └─TableRowIDScan_6(Probe)	1.00	cop[tikv]	table:t	keep order:false, stats:partial[ibb:missing, b:unInitialized]
+drop table if exists t, t1;
+create table t (j json, i bigint(20) not null primary key, key mvi((cast(j as unsigned array))));
+insert into t values ('[1,2,3]', 1);
+explain format=brief select * from t force index(mvi) where isnull(i) or json_contains(j, '1');
+id	estRows	task	access object	operator info
+IndexMerge	10.00	root		type: union
+├─TableFullScan(Build)	0.00	cop[tikv]	table:t	keep order:false, stats:pseudo
```

Comment:
It's actually a `TableDual`, the naming is wrong, and also this partial path can be dropped directly. We can improve this in the future.

Reply:
Do you mean one partial path is a table path with an empty range, and it will show like this?

Reply:
Yes. And this problem also exists for the previous non-MV index case.